public/422.html

```diff
@@ -32,9 +39,10 @@
       background-color: #efefef;
       padding-top: 25px;
       position: relative;
-      margin-top: -155px; /* negative value of footer height */
+      margin-top: -155px;
+      /* negative value of footer height */
       height: 155px;
```

Comment:
Auto-formatting caused misleading comment placement.  The previous placement seems best:

```suggestion
      margin-top: -155px; /* negative value of footer height */
      height: 155px;
```